examples/typescript/evm/baseAccount.transfer.ts

```diff
@@ -0,0 +1,36 @@
+// Usage: pnpm tsx evm/baseAccount.transfer.ts
+
+import { CdpClient, parseEther } from "@coinbase/cdp-sdk";
+import "dotenv/config";
+
+const cdp = new CdpClient();
+
+const account = await cdp.evm.getOrCreateAccount({
+  name: "Playground-Account",
+});
+
+const baseAccount = await account.useNetwork("base-sepolia");
+
+const faucetTx =
+  await baseAccount.requestFaucet({
+    token: "eth",
+});
+
+await baseAccount.waitForTransactionReceipt(faucetTx);
+
+console.log("Faucet transaction receipt:", faucetTx);
+
+const hash = await baseAccount.transfer({
+  to: "0x4252e0c9A3da5A2700e7d91cb50aEf522D0C6Fe8",
+  amount: parseEther("0.000001"),
+  token: "eth",
+  network: "base-sepolia",
```

Comment:
You can omit `network` here. Interesting that TS is not giving a compile error, will look into it